internal/cryptogen/msp/msp.go

```diff
@@ -249,7 +249,7 @@ func GenerateVerifyingMSP(
 	return nil
 }
 
-func createFolderStructure(rootDir string, local bool) error {
+func CreateFolderStructure(rootDir string, local bool) error {
```

Comment:
Not sure we should make this public just to use it for tests. Looking at the details of the function, why not just copy this as a helper function to the test file(s) you need to create your own msp?

Reply:
yep, not a problem, will make it private again.